spec/Cabin/Infotainment.vspec

```diff
@@ -140,3 +140,9 @@ HMI.DayNightMode:
   type: actuator
   allowed: ['DAY', 'NIGHT']
   description: Current display theme
+  
+HMI.SmartphoneMirroring:
+  datatype: string
+  type: sensor
+  allowed: [ 'AndoridAuto', 'AppleCarplay', 'MirrorLink', 'UNKNOWN' ]
```

Comment:
Typo: Andorid->Android

I am however not sure, this modelling is enough, as Systems might support several Mirroring Systems,. How do you express support for AppleCarPlay AND Android Auto.

Maybe better expressed by a branch and then childs like `HasMirroLink`, `HasCarPlay`, ... ?

(e.g. this https://www.dacia.co.uk/multimedia/media-nav-media-display.html supports both)

Reply:
It needs to be clarified in the description if this signal should be used to describe supported mirroring technologies and the one currently used. As Sebastian write, if it is supposed to indicate available alternatives then we may either have multiple boolean signals in a branch, or use an array here (i.e. `string[]`) to be able to select multiple ones. If it is supposed to indicate the mirroring technology currently used then I think a signal like this would work, but then I would consider replacing `UNKNOWN` with `OTHER` (as the vehicle likely knows what technology is used, but it is not one of the standard ones). (In COVESA we have so far avoided to have special values for "unknown", "undefined" or similar, based on the assumption that if a vehicle cannot report speed, mirroring-status, supported mirroring alternatives or similar due to e.g. limitations or internal errors, then it is assumed that vehicle just ignore to report/publish that signal)

But if it is to reflect current mirroring then we might need `NONE`, `INACTIVE` or similar as well to indicate that no mirror is inactive.

I would also suggest sticking to the upper case style as stated in [documentation](https://covesa.github.io/vehicle_signal_specification/rule_set/data_entry/allowed/)

i.e. something like:

`allowed: [ 'ANDROID_AUTO', 'APPLE_CARPLAY', 'MIRROR_LINK', 'UNKNOWN' ]`